public/app/core/components/Page/PageContents.tsx

```diff
@@ -6,14 +6,19 @@ import PageLoader from '../PageLoader/PageLoader';
 
 interface Props {
   isLoading?: boolean;
+  removeStyling?: boolean;
```

Comment:
is is named like a verb/action, not a boolean :) 

think better option here is pageStyle?: PageStyle  (and have a Centered + FullWidth)